save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FileUploader.kt

```diff
@@ -81,65 +83,34 @@ internal val fileIconWithMode: FC<FileIconProps> = FC { props ->
  */
 external interface UploaderProps : PropsWithChildren {
```

Comment:
we have `com.saveourtool.save.frontend.components.basic.SelectFormRequiredProps` with generic field -- maybe try using it as example?

Reply:
Ah
The point is that I like how smart casts work (after I finally removed unnecessary serialization of FileInfo)

Reply:
Forget about what I've said, now it is generic :)